src/Components/Results/index.jsx

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
 
 function Results (props) {
     return (
```

Comment:
You sent the loading prop to this file, so you can add another conditional to display a loading message when appropriate.